docs/platforms/apple/guides/ios/session-replay/performance-overhead.mdx

```diff
@@ -61,20 +61,15 @@ SentrySDK.start(configureOptions: { options in
 })
 ```
 
-### Enable Experimental View Renderer
-
-<Alert>
-  In case you are noticing issues with the experimental view renderer, please
-  report the issue on [GitHub](https://github.com/getsentry/sentry-cocoa).
-</Alert>
+### Disable View Renderer V2
 
 Starting with v8.47.0 you can enable the up-to-5x-faster new view renderer, reducing the impact of Session Replay on the main thread and potential frame drops.
+As of v8.50.0, the new view renderer is enabled by default.
```

Comment:
```suggestion
Starting with v8.50.0, the up-to-5x-faster view renderer V2 is used by default, reducing the impact of Session Replay on the main thread and potential frame drops. In previous versions (v8.47.0 and above), you can already enable it by setting `enableExperimentalViewRenderer=true`.
```

Reply:
I wanna focus the wording on the current/new state, while keeping the context of the availability as experimental feature in the last few versions